modules/standard/raid/loot_controller.py

```diff
@@ -259,31 +260,27 @@ def loot_add_item_id_cmd(self, request, _, item_id, item_count: int):
 
         return "%s was added to loot list." % item.name
 
-    @command(command="loot", params=[Const("additem", is_optional=True), Item("item"), Int("item_count", is_optional=True)],
-             description="Add an item to loot list by item_ref", access_level="all", sub_command="modify")
-    def loot_add_item_ref_cmd(self, request, _, item, item_count: int):
-        if not self.leader_controller.can_use_command(request.sender.char_id):
-            return LeaderController.NOT_LEADER_MSG
-
-        if item_count is None:
-            item_count = 1
-
-        self.add_item_to_loot(item, None, item_count)
-
-        return "%s was added to loot list." % item.name
-
     @command(command="loot", params=[Const("additem", is_optional=True), Any("item"), Int("item_count", is_optional=True)],
              description="Add an item to loot list", access_level="all", sub_command="modify")
     def loot_add_item_cmd(self, request, _, item, item_count: int):
         if not self.leader_controller.can_use_command(request.sender.char_id):
             return LeaderController.NOT_LEADER_MSG
-
+        loot = ""
         if item_count is None:
             item_count = 1
+        items = re.findall(r"(([^<]+)?<a href=[\"\']itemref://(\d+)/(\d+)/(\d+)[\"\']>([^<]+)</a>([^<]+)?)", item)
+        if items and item_count is 1:
+            for item in items:
+                if loot is not "":
+                    loot += ", " + item[0]
+                else:
+                    loot += item[0]
+                self.add_item_to_loot(item[0])
+        else:
+            loot += item
+            self.add_item_to_loot(item, None, item_count)
 
-        self.add_item_to_loot(item, None, item_count)
-
-        return "<highlight>%s<end> was added to loot list." % item
+        return "<highlight>%s<end> was added to loot list." % loot
```

Comment:
"the reason for having the different forms of the command was so that when you linked an item, it would show as an actual clickable item in the loot list instead of just a string name. this change removes that feature in order to add multiple loot at once." 
if no item count is specified (or is 1) AND at least one itemref is found, an link will be created. in the next commit, the itemlink will get "recreated"; (->   item = self.text.make_item(....) fixes an issue, when ' and " is used. (<a href="itemref://85477/21917/3'>Worn Cyber Armor Sleeves</a>) will now get correctly added as loot.